examples/plotting/file/boxplot.py

```diff
@@ -28,12 +28,9 @@ def outliers(group):
 
 # prepare outlier data for plotting, we need coordinates for every outlier.
 if not out.empty:
-    outx = []
-    outy = []
-    for keys in out.index:
-        outx.append(keys[0])
-        outy.append(out.loc[keys[0]].loc[keys[1]])
-
+    outx = list(out.index.get_level_values(0))
```

Comment:
Remove trailing whitespace to resolve code quality issues.

Reply:
You're fast! checks are still running....
White spaces already removed, waiting if more needs doing before pushing the changes.

Reply:
done